source/manual/rummager-traffic-replay.html.md

```diff
@@ -0,0 +1,35 @@
+---
+owner_slack: "#search-team"
+title: 'Replay HTTP traffic on rummager machines'
+parent: "/manual.html"
+layout: manual_layout
+section: Backups
+last_reviewed_on: 2017-09-05
+review_in: 3 months
+---
+
+Have have started using [GOR](https://github.com/buger/goreplay) to store POST and DELETED requests
```

Comment:
Have Have

Reply:
Actually this could just say "We are using", instead of started.

Reply:
Done